bin/s2nc.c

```diff
@@ -48,9 +48,6 @@ void usage()
     fprintf(stderr, "  -c [version_string]\n");
     fprintf(stderr, "  --ciphers [version_string]\n");
     fprintf(stderr, "    Set the cipher preference version string. Defaults to \"default\". See USAGE-GUIDE.md\n");
-    fprintf(stderr, "  -u [version_string]\n");
```

Comment:
Are we able to set a security policy from s2nc/d, or just the cipher prefs?

Reply:
Setting the cipher prefs would actually select a security_policy that combines all the preferences associated to it including the cipher prefs.